core/src/main/tut/README.md

```diff
@@ -14,90 +14,9 @@ Click on the demo gif below to see how PureConfig effortlessly translates your c
-  - [Unknown keys](docs/override-behaviour-for-case-classes.md#unknown-keys)
-- [Override behaviour for sealed families](docs/override-behaviour-for-sealed-families.md)
-- [Debugging implicits not found](docs/debugging-implicits-not-found.md)
-- [Error handling](docs/error-handling.md)
-- [Handling missing keys](docs/handling-missing-keys.md)
-- [Example](docs/example.md)
-- [Config files](docs/config-files.md)
-- [Support for Duration](docs/support-for-duration.md)
-- [Integrating with other libraries](#integrating-with-other-libraries)
-- [Contribute](#contribute)
-- [FAQ](docs/faq.md)
-- [License](#license)
-- [Special thanks](#special-thanks)
-
-
-## Why
-
-Loading configurations has always been a tedious and error-prone procedure. A common way to do it
-consists in writing code to deserialize each fields of the configuration. The more fields there are,
-the more code must be written (and tested and maintained...) and this must be replicated for each project.
-
-This kind of code is boilerplate because most of the times the code can be automatically generated by
-the compiler based on what must be loaded. For instance, if you are going to load an `Int` for a field
-named `foo`, then probably you want some code that gets the values associated with the key `foo` in
-the configuration and assigns it to the proper field after converting it to `Int`.
-
-The goal of this library is to create at compile-time the boilerplate necessary to load a configuration of a
-certain type. In other words, you define **what** to load and PureConfig provides **how** to load it.
-
-
-## Not yet another configuration library
-
-PureConfig is not a configuration library in the sense that it doesn't search for files or parse them.
-It can be seen as a better front-end for the existing libraries.
-It uses the [Typesafe Config][typesafe-config] library for loading raw configurations and then
-uses the raw configurations to do its magic.
-
-
-## Add PureConfig to your project
-
-In the sbt configuration file use Scala `2.10`, `2.11` or `2.12`:
-
-```scala
-scalaVersion := "2.12.4" // or "2.11.11", "2.10.6"
-```
-
-Add PureConfig to your library dependencies. For Scala `2.11` and `2.12`:
-
-```scala
-libraryDependencies ++= Seq(
-  "com.github.pureconfig" %% "pureconfig" % "0.8.0"
-)
-```
-
-For Scala `2.10` you need also the Macro Paradise plugin:
-
-```scala
-libraryDependencies ++= Seq(
-  "com.github.pureconfig" %% "pureconfig" % "0.8.0",
-  compilerPlugin("org.scalamacros" % "paradise" % "2.0.1" cross CrossVersion.patch)
-)
-```
-
-For a full example of `build.sbt` you can have a look at this [build.sbt](https://github.com/melrief/pureconfig/blob/master/example/build.sbt)
-used for the example.
-
-Earlier versions of Scala had bugs which can cause subtle compile-time problems in PureConfig.
-As a result we recommend only using the latest Scala versions within the minor series.
+Read the [documentation](https://pureconfig.github.io/setup-pureconfig.html).
```

Comment:
setup-pureconfig.html is now at https://pureconfig.github.io/docs/index.html (I thought it would also be accessible from https://pureconfig.github.io/docs/setup-pureconfig.html, but it seems not...).